website/docs/r/apigatewayv2_stage.html.markdown

```diff
@@ -56,8 +56,10 @@ Defaults to `false`. Supported only for WebSocket APIs.
 * `detailed_metrics_enabled` - (Optional) Whether detailed metrics are enabled for the default route. Defaults to `false`.
 * `logging_level` - (Optional) Logging level for the default route. Affects the log entries pushed to Amazon CloudWatch Logs.
 Valid values: `ERROR`, `INFO`, `OFF`. Defaults to `OFF`. Supported only for WebSocket APIs. Terraform will only perform drift detection of its value when present in a configuration.
-* `throttling_burst_limit` - (Optional) Throttling burst limit for the default route.
-* `throttling_rate_limit` - (Optional) Throttling rate limit for the default route.
+* `throttling_burst_limit` - (Optional) Throttling burst limit for the default route. Defaults to `0`.
```

Comment:
I'm not sure that I agree with this change. Typically in our resource documentation, we only mention a default value if the default value is coming from the resource schema. In this case, [a default value is not specified](https://github.com/hashicorp/terraform-provider-aws/blob/c6cef71fa1c4cd785a9aae33c448bbcf4273a8e6/internal/service/apigatewayv2/stage.go#L108-L111) in the resource definition.